meilisearch/index.py

```diff
@@ -137,7 +143,11 @@ def get_tasks(self) -> Dict[str, List[Dict[str, Any]]]:
         MeiliSearchApiError
             An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://docs.meilisearch.com/errors/#meilisearch-errors
         """
-        return get_tasks(self.config, self.uid)
+        if parameters is not None:
+            parameters.setdefault('indexUid', []).append(self.uid)
```

Comment:
What if you do
`index.getTasks({})` then parameters is not None but it does not contain indexUid